### PR TITLE
85 logo animation

### PR DESCRIPTION
**Pull request by bohnerjosh**
Added Logo Animation to app.

Implemented as its own screen that is navigated to upon login.

### CODE REVIEW

File: src/Widgets/AnimatedLogo/AnimatedLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//@ts-ignore - the file exists
+import PEW_35_ANIMATED_LOGO from '../../../assets/Mobile-Persist-600px-GIF.gif';
+import { ROUTE_NAMES } from "../../TypesAndInterfaces/routes";
+import { StackNavigationProps } from "../../TypesAndInterfaces/custom-types";
+import { StackActions } from '@react-navigation/native';
+
+export const AnimatedLogo = (props:StackNavigationProps):JSX.Element => {
+
+    const [timeoutID, setTimeoutID] = useState<NodeJS.Timeout>(setTimeout(() => props.navigation.dispatch(StackActions.replace(ROUTE_NAMES.BOTTOM_TAB_NAVIGATOR_ROUTE_NAME)), 9000));
+
```
Comment (ethanjohnsrud) [inline src/Widgets/AnimatedLogo/AnimatedLogo.tsx:13]: Do we want to be a little tighter on the timing?
I timed the animation to be 5.6 seconds, so maybe a little past that.

Also is it possible to do a fade transition?  In the web I reduced the page opacity to zero, then swapped routes.

I like the tap through, that'll be helpful in by debugging.

**Review by ethanjohnsrud — CHANGES_REQUESTED** [review]
Just a few ideas, let me know what you think.

File: src/Widgets/AnimatedLogo/AnimatedLogo.tsx (hunk 1 above)
Comment (bohnerjosh) [reply src/Widgets/AnimatedLogo/AnimatedLogo.tsx:13]: Changed timeout length to 7 seconds, which is the same as the portal. Also added a smooth fade transition.

Comment (ethanjohnsrud) [reply src/Widgets/AnimatedLogo/AnimatedLogo.tsx:13]: Perfect, thanks Man

**Review by ethanjohnsrud — APPROVED** [review]
Appreciate the changes, merge in which ever order you'd like.

**Pull request merged**